app/src/main/java/com/github/se/icebreakrr/model/message/MeetingRequestViewModel.kt

```diff
@@ -96,60 +94,46 @@ class MeetingRequestViewModel(
    *
    * @param newToken the current token of the user to which we want to send the message
    */
-  fun onLocalTokenChange(newToken: String) {
+  fun setTargetToken(newToken: String) {
     meetingRequestState = meetingRequestState.copy(targetToken = newToken)
   }
   /**
    * Sets the message of the meeting request
    *
    * @param newMessage the message we want to send
    */
-  fun onMeetingRequestChange(newMessage: String) {
-    meetingRequestState = meetingRequestState.copy(message = newMessage)
+  fun setMeetingRequestChangeFirstMessage(message1: String) {
```

Comment:
You should find different names for the two types of messages instead of message1 and message2 for clarity purpose.

Reply:
I could keep one called message and the other locationMessage, to make minimal changes in the firebase cloud functions and to make things clearer